kaal-presentation/src/main/kotlin/cz/eman/kaal/presentation/adapter/BindingAdapters.kt

```diff
@@ -28,13 +28,14 @@ fun <T : Any> bindRecyclerView(
     clickListener: ((View, T) -> Unit)?,
     longClickListener: ((View, T) -> Unit)?,
     differ: DiffUtil.ItemCallback<T>?,
-    limit: Int?
+    limit: Int?,
+    onComplete: ((RecyclerView, RecyclerView.Adapter<*>) -> Unit)?
```

Comment:
The `RecyclerView.Adapter` can be obtained `RecyclerView.adapter`, why provide it as a second parameter?